tests/constraints/tests.py

```diff
@@ -995,6 +995,42 @@ def test_validate_expression_str(self):
             exclude={"name"},
         )
 
+    def test_validate_nullable_textfield_with_isnull_true(self):
```

Comment:
Does this test need `@skipUnlessDBFeature("supports_table_check_constraints")`? It's a required_db_feature for the `Product` table.

Reply:
Good catch :dart:

Reply:
Check out #17356.